MMR.Randomizer/Models/Rom/MMFile.cs

```diff
@@ -12,6 +12,7 @@ public class MMFile
         public bool IsCompressed;
         public bool WasEdited;
         public bool IsStatic;
+        public bool HardRomAddr;
```

Comment:
Can't find where this is used.

Reply:
Left over from development, an alternative was found.

Removed.